web/satellite/src/components/account/DeleteAccountPopup.vue

```diff
@@ -24,8 +24,8 @@
                     <Button label='Delete' width='205px' height='48px' class='red' :onPress='onDeleteAccountClick'/>
                 </div>
             </div>
-            <div class='delete-account__close-cross-container'>
-                <svg width='16' height='16' viewBox='0 0 16 16' fill='none' xmlns='http://www.w3.org/2000/svg' v-on:click='onCloseClick'>
+            <div class='delete-account__close-cross-container' @click='onCloseClick'>
```

Comment:
Just curious what's the convention for quotes in our templates. I have seen both double quotes and single quotes in the codebase

Reply:
well, in typescript we use only single quotes. I think we should decide which one to use in templates or make it single everywhere to follow code consistency. We can do it on our refactoring session